packages/react/src/analytics/integrations/GA4/types/index.ts

```diff
@@ -48,7 +48,10 @@ export type OnPreProcessCommandsHandler = (
 ) => GA4CommandList;
 
 export type GA4CommandList = GA4Command[];
-export type GA4Command = [command: string, ...args: unknown[]];
+export type GA4Command = [
+  command: string,
+  ...args: unknown[] | Record<string, string>[],
```

Comment:
I think we should continue use the unknown here as it is not certain that all commands receive a `Record<string, string>`. Ideally, we could type this differently for the `event` and `config` commands but I believe for this case we can let it as is. I will suggest another change where this is being used in the code.